ansible/roles/ocp-client-vm/tasks/main.yml

```diff
@@ -287,9 +302,9 @@
       path: "{{ item.directory }}/.bash-git-prompt"
       owner: "{{ item.user }}"
       group: "{{ item.group }}"
-      recurse: yes
+      recurse: true
     loop:
-    - { directory: "/home/{{ student_name }}",  user: "{{ student_name }}",  group: "users" }
+    - {directory: "/home/{{ student_name }}", user: "{{ student_name }}", group: "users"}
```

Comment:
and here... this is a loop with one item? How weird, this should not be a loop at all, should just be:
```
  - name: Change ownership of bash-git-prompt for student_name
    file:
      path: /home/{{ student_name }}/.bash-git-prompt
      owner: "{{ student_name }}"
      group: users
      recurse: true
```